main.py

```diff
@@ -141,6 +141,24 @@ async def submit_experiment(
     return remote.submit(pipeline, expid, priority, due_date, False)
 
 
+@app.get("/experiment/running/")
+async def get_running_experiment() -> Optional[int]:
+    """Gets the running experiment RID.
+
+    It assumes that there is at most one running experiment.
+    
+    Returns:
+        The run identifier of the running experiment.
+        If no running experiment, it returns None.
+    """
+    remote = get_client("master_schedule")
+    status = remote.get_status()
+    try:
+        return next(filter(lambda rid: status[rid]["status"] == "running", status))
```

Comment:
If you have to use `lambda` in `filter()`, it is often more intuitive to use comprehension:
```python
next(rid for rid, info in status.items() if info["status"] == "running")
```

In addition, if you don't want to use `try-except` here, you can just use `for`:
```python
for rid, info in status.items():
    if info["status"] == "running":
        return rid
return None
```

Reply:
Oh, the second one looks much better! I will apply it.